lib/version.h

```diff
@@ -5,17 +5,17 @@
 #define versionH
 
 #define CPPCHECK_MAJOR_VERSION 2
-#define CPPCHECK_MINOR_VERSION 12
-#define CPPCHECK_DEVMINOR_VERSION 13
+#define CPPCHECK_MINOR_VERSION 13
+#define CPPCHECK_BUGFIX_VERSION 99
 #define CPPCHECK_FIX_VERSION 0
 
 #define STRINGIFY(x) STRING(x)
 #define STRING(VER) #VER
-#if CPPCHECK_MINOR_VERSION == CPPCHECK_DEVMINOR_VERSION
+#if CPPCHECK_BUGFIX_VERSION < 99
 #define CPPCHECK_VERSION_STRING STRINGIFY(CPPCHECK_MAJOR_VERSION) "." STRINGIFY(CPPCHECK_MINOR_VERSION) "." STRINGIFY(CPPCHECK_FIX_VERSION)
 #define CPPCHECK_VERSION CPPCHECK_MAJOR_VERSION,CPPCHECK_MINOR_VERSION,CPPCHECK_FIX_VERSION,0
 #else
-#define CPPCHECK_VERSION_STRING STRINGIFY(CPPCHECK_MAJOR_VERSION) "." STRINGIFY(CPPCHECK_DEVMINOR_VERSION) " dev"
```

Comment:
Sorry but the CPPCHECK_VERSION_STRING will be messed up after these changes... both for debug and release versions.

So either if you can fix it somehow. Otherwise please change the code back so we have "DEVMINOR" again. As I understand, then "MINOR" should be 13 and "DEVMINOR" should be 14.

Reply:
can you compile and test this? What does cppcheck --version say? I assume that it will be something like "2.13+1 dev" instead of "2.14 dev".

Reply:
Even worse, it says `Cppcheck 2.99 dev`...